eng/DownloadSharedSource.ps1

```diff
@@ -14,7 +14,7 @@ function DownloadAll([string[]]$files, [string]$baseUrl, [string]$downloadPath)
 
 $downloadPath = Resolve-Path (Join-Path $PSScriptRoot '..' 'src' 'assets' 'Azure.Core.Shared')
 Get-ChildItem $downloadPath -Filter *.cs | Remove-Item;
-$files = @('ClientDiagnostics.cs', 'DiagnosticScope.cs', 'DiagnosticScopeFactory.cs', 'ContentTypeUtilities.cs', 'HttpMessageSanitizer.cs',
```

Comment:
Files in `Azure.Core.Shared` folder are updated from azure-sdk-for-net repo after the PR https://github.com/Azure/azure-sdk-for-net/pull/28375